camptix.php

```diff
@@ -470,9 +470,18 @@ function enqueue_scripts() {
 			'enterEmail' => __( 'Please enter the e-mail addresses in the forms above.', 'camptix' ),
 			'ajaxURL'    => admin_url( 'admin-ajax.php' ),
 		) );
-
 		// Let's play by the rules and print this in the <head> section.
 		wp_enqueue_style( 'camptix' );
+
+
+		if ( (bool) $this->options['select2_enabled'] ) {
+			wp_register_style( 'select2', plugins_url( '/external/select2/css/select2.min.css', __FILE__ ), array(), $this->version );
+			wp_register_script( 'select2', plugins_url( '/external/select2/js/select2.min.js', __FILE__ ), array(), $this->version );
+
+			wp_enqueue_style( 'select2' );
```

Comment:
Just noting here at that since the Select2 assets will always be both registered and enqueued if the option is enabled, you can save a step by just calling the `enqueue` functions with all the same parameters, and they will take care of the registration as well.

But I guess another question is, should the assets _always_ be enqueued, or only for certain views?